config/scrapper_config_example.py

```diff
@@ -0,0 +1,15 @@
+# Message broker server URL. You can use any broker,
+# such as RabbitMQ or Redis.
+# This is an example of RabbitMQ default URL.
+MESSAGE_BROKER_URL = 'amqp://guest:guest@localhost//'
```

Comment:
I think it won't work, because in the end of url //

Reply:
'//' in this URI specify a virtual host by default

Reply:
Also, I'd put {username}:{password}@{ip} in config file